.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[build]
+rustflags = ["--cfg", "tokio_unstable"]
```

Comment:
to make [`tokio-console` work:](https://github.com/tokio-rs/console?tab=readme-ov-file#using-it):
> in order to collect task data from Tokio, the tokio_unstable cfg must be enabled.

Reply:
With the Rust stable version, there hasn't been any problems. The `tokio-console` command displays just fine, although I was not able to see the [histogram](https://github.com/tokio-rs/console/blob/main/assets/readme/task-details.png).

The worst it could do would be returning a blank view.

Reply:
This doesn't work for me, unfortunately I have to explicitly pass the `RUSTFLAGS="--cfg tokio_unstable"` when trying to run or build the client. 

I assume it's because I do have a `~/.cargo/config.toml` file where I already specify some rust flags
```
[target.aarch64-apple-darwin]
rustflags = ["-C", "link-arg=-fuse-ld=/opt/homebrew/opt/llvm/bin/ld64.lld"]
```
and this might override the workspace configuration. 

I confirmed this by removing my 'global' config.toml and then recompiling. This worked.

Generally I would prefer if we could enable this `tokio-console` thing with an extra feature flag and have it disabled by default. Do you think this would be possible @b-yap?

Reply:
See https://github.com/pendulum-chain/spacewalk/pull/517/commits/e77d7acb7598b6beb300dffcae085c9667e53777

I tried to set it through `[env]` .cargo/config.toml, and it didn't work. All `RUSTFLAGS`, `CARGO_ENCODED_RUSTFLAGS` and `CARGO_BUILD_FLAGS` won't work.


I also tried to do it through a build script and didn't work:
```rust
fn main(){
    cfg_if::cfg_if! {
        if #[cfg(feature = "allow-debugger")] {
            println!("cargo:rustc-env=RUSTFLAGS=\"--cfg tokio_unstable\"");
        }
    }
}
```
and
`println!("cargo:rustc-flags=\"--cfg tokio_unstable\"");`
as explained in https://github.com/rust-lang/cargo/pull/10141.


In the end I updated the doc to tell the user to set the rustflags themselves.